tests/models/test_ip_adapters.py

```diff
@@ -0,0 +1,414 @@
+# coding=utf-8
```

Comment:
move file to `tests/pipelines/ip_adapters/test_ip_adapter_stable_diffusion.py`

Reply:
And add tests to `tests/models/test_unet_2d_condition.py` that just concern unet code